src/utils/classNames.ts

```diff
@@ -0,0 +1,2 @@
+export const classNames = (...classes: (false | null | undefined | string)[]): string =>
+    classes.filter(Boolean).join(" ");
```

Comment:
There is a very neat library called `classnames` (https://www.npmjs.com/package/classnames) with lots of cool features that we can use for this. However, considering the little amount of time we have left until deployment, we can refactor this later :)

Reply:
Thank you, didn't know about it